modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -0,0 +1,384 @@
+        default: true
+        credential:
+          key: cloud
+          name: <default_secret> <9>
+        objectStorage:
+          bucket: <bucket_name> <10>
+          prefix: <prefix> <11>
+----
+<1> The `kubevirt` plugin is mandatory for {VirtProductName}.
+<2> Specify the plugin for the backup provider, for example, `gcp`, if it exists.
+<3> The `csi` plugin is mandatory for backing up PVs with CSI snapshots. The `csi` plugin uses the link:https://{velero-domain}/docs/main/csi/[Velero CSI beta snapshot APIs]. You do not need to configure a snapshot location.
+<4> The `openshift` plugin is mandatory.
+<5> Specify how many minutes to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. The default is 10m.
+<6> The administrative agent that routes the administrative requests to servers.
+<7> Set this value to `true` if you want to enable `nodeAgent` and perform File System Backup.
+<8> Enter `kopia` or `restic` as your uploader. You cannot change the selection after the installation. The `nodeAagent` deploys a daemon set, which means that the `nodeagent` pods run on each working node. You can configure File System Backup by adding `spec.defaultVolumesToFsBackup: true` to the `Backup` CR.
+<9> Specify the nodes on which Kopia or Restic are available. By default, Kopia or Restic run on all nodes.
+<10> Specify the backup provider.
+<11> Specify the correct default name for the `Secret`, for example, `cloud-credentials-gcp`, if you use a default plugin for the backup provider. If specifying a custom name, then the custom name is used for the backup location. If you do not specify a `Secret` name, the default name is used.
+<12> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
+<13> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
+endif::[]
+
+. Click *Create*.
+. Verify the installation by viewing the OADP resources:
++
+[source,terminal]
+----
+$ oc get all -n openshift-adp
+----
++
+.Example output
++
+----
+NAME                                                     READY   STATUS    RESTARTS   AGE
+pod/oadp-operator-controller-manager-67d9494d47-6l8z8    2/2     Running   0          2m8s
+pod/restic-9cq4q                                         1/1     Running   0          94s
+pod/restic-m4lts                                         1/1     Running   0          94s
+pod/restic-pv4kr                                         1/1     Running   0          95s
+pod/velero-588db7f655-n842v                              1/1     Running   0          95s
+
+NAME                                                       TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+service/oadp-operator-controller-manager-metrics-service   ClusterIP   172.30.70.140    <none>        8443/TCP   2m8s
+
```

Comment:
```suggestion
service/openshift-adp-velero-metrics-svc                   ClusterIP   172.30.10.0      <none>        8085/TCP   8h

```

Reply:
@stillalearner 
Do you want both of these?

NAME                                                       TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
service/oadp-operator-controller-manager-metrics-service   ClusterIP   172.30.70.140    <none>        8443/TCP   2m8s
service/openshift-adp-velero-metrics-svc                   ClusterIP   172.30.10.0      <none>        8085/TCP   8h

Reply:
yes, both should be there, after installing the DPA.